### DISCUSSION TITLE
Inaccessible Account

### DISCUSSION THREAD

**Discussion by kiko-g** [planetscale/discussion#426] [Bug Reports] [answered]
I think my account was not successfully deleted and I cannot login, forget password (email doesn't reach my inbox) or create a new account with the same email or Github account. This might be a bug. What can I do?

**Comment by jgreet** [comment, marked as answer]
Hi @kiko-g, we removed our record of your old account. You should be able to sign up again with the same email address or GitHub ID now.

**Reply by ggfer** [reply]
Hi @jgreet,
same issue as kiko-g. can you help me with my current github account(primary email: jojowhoooo@gmail.com).

Thank you!

**Reply by mscoutermarsh** [reply]
@jojowhoooo Could you please send a request here: https://support.planetscale.com/hc/en-us (more reliable in case someone misses a GitHub notification)

**Reply by jgreet** [reply]
Handled this in Zendesk yesterday 👍
